services-camel/src/main/java/edu/unc/lib/dl/services/camel/longleaf/LongleafRouter.java

```diff
@@ -28,6 +28,8 @@
  * @author bbpennel
  */
 public class LongleafRouter extends RouteBuilder {
+    @BeanInject(value = "getUrisProcessor")
+    private GetUrisProcessor getUrisProcessor;
```

Comment:
Up to you if you want to change it, but `GetUrisProcessor` doesn't have any dependencies and is never mocked, so it'd be fine to initialize it directly in this class, either in the constructor or in the configure method.